.github/workflows/reusable_fast.yml

```diff
@@ -1,76 +1,58 @@
-# Fast builds
 name: FastBuild
 
 on: workflow_call
 
 permissions:
   contents: read
+  packages: read
 
 env:
   BUILD_DIR : "${{github.workspace}}/build"
   INSTL_DIR : "${{github.workspace}}/../install-dir"
 
 jobs:
-  FastBuild:
-    name: Fast builds
+  FastBuild_Linux:
+    name: Fast builds (Linux)
+    runs-on: ${{ matrix.os }}
+    container:
+      image: ${{ matrix.os == 'ubuntu-latest' && 'ghcr.io/rbanka1/umf2-ubuntu-22.04:latest' || matrix.os == 'ubuntu-20.04' && 'ghcr.io/rbanka1/umf2-ubuntu-20.04:latest' }}
+      options: --user root --privileged
+      volumes:
+        - /home/runner/work/unified-memory-framework/unified-memory-framework:/home/runner/work/unified-memory-framework/unified-memory-framework
+
     env:
       VCPKG_PATH: "${{github.workspace}}/build/vcpkg/packages/hwloc_x64-windows;${{github.workspace}}/build/vcpkg/packages/tbb_x64-windows;${{github.workspace}}/build/vcpkg/packages/jemalloc_x64-windows"
     strategy:
       matrix:
         include:
-          - os: windows-latest
-            build_tests: 'ON'
-            simple_cmake: 'OFF'
-          # pure C build (Windows)
-          - os: windows-latest
-            # Tests' building is off for a pure C build
-            build_tests: 'OFF'
-            simple_cmake: 'OFF'
           - os: ubuntu-latest
+            disjoint: 'ON'
```

Comment:
disjoint parameter was removed in https://github.com/oneapi-src/unified-memory-framework/pull/898 - it's not needed anymore; our CMake no longer has `UMF_BUILD_LIBUMF_POOL_DISJOINT` option.